native/asgardeo/src/main/java/io/crypto/Decryption.java

```diff
@@ -0,0 +1,27 @@
+package io.crypto;
+
+import static io.crypto.Utils.getPrivateKey;
+import static io.crypto.Utils.valueToEmptyOrToString;
+
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.creators.ErrorCreator;
+
+public class Decryption {
+
+    public static Object decrypt(Object encryptedString, Object decryptionKey, Object algo) {
```

Comment:
Let's add a doc comment to this function

Reply:
Updated as suggested